src/files/GzFile.cpp

```diff
@@ -34,7 +34,8 @@ GzFile::GzFile(const std::string& path, File::Mode mode): TextFileImpl(path) {
         openmode = "wb7";
         break;
     case File::APPEND:
-        throw file_error("appending (open mode 'a') is not supported with gziped files");
+        openmode = "ab";
```

Comment:
We may want to open the file in 'ab7' mode, to ensure the same compression ratio is used as in writing mode.

Reply:
Makes good sense.